pkg/operator/handler.go

```diff
@@ -305,6 +305,14 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 		if err != nil {
 			return err
 		}
+		svc := event.Object.(*coreapi.Service)
+		if cr != nil {
+			svcHostname := fmt.Sprintf("%s.%s.cluster.svc:%d", svc.Name, svc.Namespace, svc.Spec.Ports[0].Port)
```

Comment:
`.cluster.svc`? Is it something new in 4.0?

Reply:
@bparees you mean `my-svc.my-namespace.svc.cluster.local` ?

https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services

Reply:
sorry yes i got it backwards.

Reply:
pushed fix.